nova/scheduler/filter_scheduler.py

```diff
@@ -298,7 +298,7 @@ def _ensure_sufficient_hosts(self, context, hosts, required_count,
                   '%(required_count)d instances requested to build.',
                   {'hosts': len(hosts),
                    'required_count': required_count})
-        reason = _('There are not enough hosts available.')
+        reason = _('There are not enough hosts available. To trouble-shoot, please see http://bit.ly/faq-instance-failure')
```

Comment:
```suggestion
        reason = _('There are not enough hosts available. To troubleshoot, please see bit.ly/faq-instance-failure')
```

Reply:
Just trying to save space, not sure the protocol is important for this interface.